dashboard/app/models/levels/level_group.rb

```diff
@@ -74,11 +74,11 @@ def initialize(page_number, levels_and_texts_offset, levels_and_texts, levels_of
     attr_reader :levels_offset
 
     def levels
-      levels_and_texts.reject {|l| l.is_a?(External)}
+      levels_and_texts&.reject {|l| l.is_a?(External)}
     end
 
     def texts
-      levels_and_texts.select {|l| l.is_a?(External)}
+      levels_and_texts&.select {|l| l.is_a?(External)}
```

Comment:
merging this makes me a little uncomfortable -- we have caught a number of issues where `levels_and_texts` is nil because it exposes that there is someplace we haven't updated our system to use the new level group format. @dmcavoy  can you describe in more detail what was failing, maybe by pointing to a stack trace?

Reply:
Created PR to fix: https://github.com/code-dot-org/code-dot-org/pull/36559